<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />

<meta name="viewport" content="width=device-width, initial-scale=1" />

<meta name="author" content="Zofia Wicik" />

<meta name="date" content="2020-08-31" />

<title>vignette-wizbionet</title>

<script>// Hide empty <a> tag within highlighted CodeBlock for screen reader accessibility (see https://github.com/jgm/pandoc/issues/6352#issuecomment-626106786) -->
// v0.0.1
// Written by JooYoung Seo (jooyoung@psu.edu) and Atsushi Yasumoto on June 1st, 2020.

document.addEventListener('DOMContentLoaded', function() {
  const codeList = document.getElementsByClassName("sourceCode");
  for (var i = 0; i < codeList.length; i++) {
    var linkList = codeList[i].getElementsByTagName('a');
    for (var j = 0; j < linkList.length; j++) {
      if (linkList[j].innerHTML === "") {
        linkList[j].setAttribute('aria-hidden', 'true');
      }
    }
  }
});
</script>
<script>$(document).ready(function(){
    if (typeof $('[data-toggle="tooltip"]').tooltip === 'function') {
        $('[data-toggle="tooltip"]').tooltip();
    }
    if ($('[data-toggle="popover"]').popover === 'function') {
        $('[data-toggle="popover"]').popover();
    }
});
</script>
<style type="text/css">
.lightable-minimal {
border-collapse: separate;
border-spacing: 16px 1px;
width: 100%;
margin-bottom: 10px;
}
.lightable-minimal td {
margin-left: 5px;
margin-right: 5px;
}
.lightable-minimal th {
margin-left: 5px;
margin-right: 5px;
}
.lightable-minimal thead tr:last-child th {
border-bottom: 2px solid #00000050;
empty-cells: hide;
}
.lightable-minimal tbody tr:first-child td {
padding-top: 0.5em;
}
.lightable-minimal.lightable-hover tbody tr:hover {
background-color: #f5f5f5;
}
.lightable-minimal.lightable-striped tbody tr:nth-child(even) {
background-color: #f5f5f5;
}
.lightable-classic {
border-top: 0.16em solid #111111;
width: 100%;
margin-bottom: 10px;
margin: 10px 5px;
}
.lightable-classic caption {
color: #222222;
}
.lightable-classic td {
padding-left: 5px;
padding-right: 5px;
color: #222222;
}
.lightable-classic th {
padding-left: 5px;
padding-right: 5px;
font-weight: normal;
color: #222222;
}
.lightable-classic thead tr:last-child th {
border-bottom: 0.10em solid #111111;
}
.lightable-classic tbody tr:last-child td {
border-bottom: 0.14em solid #111111;
}
.lightable-classic.lightable-hover tbody tr:hover {
background-color: #F9EEC1;
}
.lightable-classic.lightable-striped tbody tr:nth-child(even) {
background-color: #f5f5f5;
}
.lightable-classic-2 {
border-top: 3px double #111111;
width: 100%;
margin-bottom: 10px;
}
.lightable-classic-2 caption {
color: #222222;
}
.lightable-classic-2 td {
padding-left: 5px;
padding-right: 5px;
color: #222222;
}
.lightable-classic-2 th {
padding-left: 5px;
padding-right: 5px;
font-weight: normal;
color: #222222;
}
.lightable-classic-2 tbody tr:last-child td {
border-bottom: 3px double #111111;
}
.lightable-classic-2 thead tr:last-child th {
border-bottom: 1px solid #111111;
}
.lightable-classic-2.lightable-hover tbody tr:hover {
background-color: #F9EEC1;
}
.lightable-classic-2.lightable-striped tbody tr:nth-child(even) {
background-color: #f5f5f5;
}
.lightable-material {
min-width: 100%;
white-space: nowrap;
table-layout: fixed;
font-family: Roboto, sans-serif;
border: 1px solid #EEE;
border-collapse: collapse;
margin-bottom: 10px;
}
.lightable-material th {
height: 56px;
padding-left: 16px;
padding-right: 16px;
}
.lightable-material td {
height: 52px;
padding-left: 16px;
padding-right: 16px;
border-top: 1px solid #eeeeee;
}
.lightable-material.lightable-hover tbody tr:hover {
background-color: #f5f5f5;
}
.lightable-material.lightable-striped tbody tr:nth-child(even) {
background-color: #f5f5f5;
}
.lightable-material.lightable-striped tbody td {
border: 0;
}
.lightable-material.lightable-striped thead tr:last-child th {
border-bottom: 1px solid #ddd;
}
.lightable-material-dark {
min-width: 100%;
white-space: nowrap;
table-layout: fixed;
font-family: Roboto, sans-serif;
border: 1px solid #FFFFFF12;
border-collapse: collapse;
margin-bottom: 10px;
background-color: #363640;
}
.lightable-material-dark th {
height: 56px;
padding-left: 16px;
padding-right: 16px;
color: #FFFFFF60;
}
.lightable-material-dark td {
height: 52px;
padding-left: 16px;
padding-right: 16px;
color: #FFFFFF;
border-top: 1px solid #FFFFFF12;
}
.lightable-material-dark.lightable-hover tbody tr:hover {
background-color: #FFFFFF12;
}
.lightable-material-dark.lightable-striped tbody tr:nth-child(even) {
background-color: #FFFFFF12;
}
.lightable-material-dark.lightable-striped tbody td {
border: 0;
}
.lightable-material-dark.lightable-striped thead tr:last-child th {
border-bottom: 1px solid #FFFFFF12;
}
.lightable-paper {
width: 100%;
margin-bottom: 10px;
color: #444;
}
.lightable-paper thead tr:last-child th {
color: #666;
vertical-align: bottom;
border-bottom: 1px solid #00000020;
line-height: 1.15em;
padding: 10px 5px;
}
.lightable-paper td {
vertical-align: middle;
border-bottom: 1px solid #00000010;
line-height: 1.15em;
padding: 7px 5px;
}
.lightable-paper.lightable-hover tbody tr:hover {
background-color: #F9EEC1;
}
.lightable-paper.lightable-striped tbody tr:nth-child(even) {
background-color: #00000008;
}
.lightable-paper.lightable-striped tbody td {
border: 0;
}
</style>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>



<style type="text/css">body {
background-color: #fff;
margin: 1em auto;
max-width: 700px;
overflow: visible;
padding-left: 2em;
padding-right: 2em;
font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.35;
}
#TOC {
clear: both;
margin: 0 0 10px 10px;
padding: 4px;
width: 400px;
border: 1px solid #CCCCCC;
border-radius: 5px;
background-color: #f6f6f6;
font-size: 13px;
line-height: 1.3;
}
#TOC .toctitle {
font-weight: bold;
font-size: 15px;
margin-left: 5px;
}
#TOC ul {
padding-left: 40px;
margin-left: -1.5em;
margin-top: 5px;
margin-bottom: 5px;
}
#TOC ul ul {
margin-left: -2em;
}
#TOC li {
line-height: 16px;
}
table {
margin: 1em auto;
border-width: 1px;
border-color: #DDDDDD;
border-style: outset;
border-collapse: collapse;
}
table th {
border-width: 2px;
padding: 5px;
border-style: inset;
}
table td {
border-width: 1px;
border-style: inset;
line-height: 18px;
padding: 5px 5px;
}
table, table th, table td {
border-left-style: none;
border-right-style: none;
}
table thead, table tr.even {
background-color: #f7f7f7;
}
p {
margin: 0.5em 0;
}
blockquote {
background-color: #f6f6f6;
padding: 0.25em 0.75em;
}
hr {
border-style: solid;
border: none;
border-top: 1px solid #777;
margin: 28px 0;
}
dl {
margin-left: 0;
}
dl dd {
margin-bottom: 13px;
margin-left: 13px;
}
dl dt {
font-weight: bold;
}
ul {
margin-top: 0;
}
ul li {
list-style: circle outside;
}
ul ul {
margin-bottom: 0;
}
pre, code {
background-color: #f7f7f7;
border-radius: 3px;
color: #333;
white-space: pre-wrap; 
}
pre {
border-radius: 3px;
margin: 5px 0px 10px 0px;
padding: 10px;
}
pre:not([class]) {
background-color: #f7f7f7;
}
code {
font-family: Consolas, Monaco, 'Courier New', monospace;
font-size: 85%;
}
p > code, li > code {
padding: 2px 0px;
}
div.figure {
text-align: center;
}
img {
background-color: #FFFFFF;
padding: 2px;
border: 1px solid #DDDDDD;
border-radius: 3px;
border: 1px solid #CCCCCC;
margin: 0 5px;
}
h1 {
margin-top: 0;
font-size: 35px;
line-height: 40px;
}
h2 {
border-bottom: 4px solid #f7f7f7;
padding-top: 10px;
padding-bottom: 2px;
font-size: 145%;
}
h3 {
border-bottom: 2px solid #f7f7f7;
padding-top: 10px;
font-size: 120%;
}
h4 {
border-bottom: 1px solid #f7f7f7;
margin-left: 8px;
font-size: 105%;
}
h5, h6 {
border-bottom: 1px solid #ccc;
font-size: 105%;
}
a {
color: #0033dd;
text-decoration: none;
}
a:hover {
color: #6666ff; }
a:visited {
color: #800080; }
a:visited:hover {
color: #BB00BB; }
a[href^="http:"] {
text-decoration: underline; }
a[href^="https:"] {
text-decoration: underline; }

code > span.kw { color: #555; font-weight: bold; } 
code > span.dt { color: #902000; } 
code > span.dv { color: #40a070; } 
code > span.bn { color: #d14; } 
code > span.fl { color: #d14; } 
code > span.ch { color: #d14; } 
code > span.st { color: #d14; } 
code > span.co { color: #888888; font-style: italic; } 
code > span.ot { color: #007020; } 
code > span.al { color: #ff0000; font-weight: bold; } 
code > span.fu { color: #900; font-weight: bold; } 
code > span.er { color: #a61717; background-color: #e3d2d2; } 
</style>




</head>

<body>




<h1 class="title toc-ignore">vignette-wizbionet</h1>
<h4 class="author">Zofia Wicik</h4>
<h4 class="date">2020-08-31</h4>



<!-- .markdown-block { -->
<!--    padding-top: 0 !important; -->
<!--    padding-bottom: 0 !important; -->
<!-- } -->
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>This package was build to provide set of tools enabling identification of the top genes and non-coding RNAs (miRNAs, lncRNAs) from expression studies. It can also be applied to arbitrary selected lists of genes mined from the literature.</p>
<p>In order to identify best coding and non-coding genes that would serve as key regulators of complex diseases, we developed a key-gene scoring system taking into account their mutual regulation. In this document we we will try to show examples of analyses which can be made with this toolbox:</p>
</div>
<div id="installation" class="section level2">
<h2>Installation</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="co">#Some of the dependencies are not downloaded automatically yet. </span></span>
<span id="cb1-2"><a href="#cb1-2"></a><span class="co">#Below is the code which will install all of them. Just copy it and paste to your R console</span></span>
<span id="cb1-3"><a href="#cb1-3"></a><span class="co">#dependencies</span></span>
<span id="cb1-4"><a href="#cb1-4"></a>      <span class="co">#dplyr (&gt;= 1.0.2), multiMiR (&gt;= 1.10.0),  stringr (&gt;= 1.4.0),  XML (&gt;= 3.99-0.5), </span></span>
<span id="cb1-5"><a href="#cb1-5"></a>      <span class="co">#OneR (&gt;= 2.2.0), plyr (&gt;= 1.8.6), tidyselect (&gt;= 1.1.0), kableExtra (&gt;= 1.1.0)</span></span>
<span id="cb1-6"><a href="#cb1-6"></a></span>
<span id="cb1-7"><a href="#cb1-7"></a></span>
<span id="cb1-8"><a href="#cb1-8"></a><span class="co"># 1. installation of the necessary packages, if it will not work download tar file from the webpage.If will not work download tar files and install them using command #   # </span></span>
<span id="cb1-9"><a href="#cb1-9"></a>    </span>
<span id="cb1-10"><a href="#cb1-10"></a>    <span class="co">#install.packages(&quot;path_to_the package&quot;) </span></span>
<span id="cb1-11"><a href="#cb1-11"></a>      <span class="cf">if</span> (<span class="op">!</span><span class="kw">requireNamespace</span>(<span class="st">&quot;BiocManager&quot;</span>, <span class="dt">quietly=</span><span class="ot">TRUE</span>))</span>
<span id="cb1-12"><a href="#cb1-12"></a>          <span class="kw">install.packages</span>(<span class="st">&quot;BiocManager&quot;</span>)</span>
<span id="cb1-13"><a href="#cb1-13"></a>      BiocManager<span class="op">::</span><span class="kw">install</span>()</span>
<span id="cb1-14"><a href="#cb1-14"></a>      BiocManager<span class="op">::</span><span class="kw">install</span>(<span class="st">&quot;multiMiR&quot;</span>)</span>
<span id="cb1-15"><a href="#cb1-15"></a>    </span>
<span id="cb1-16"><a href="#cb1-16"></a>    <span class="co"># https://cran.r-project.org/web/packages/OneR/index.html</span></span>
<span id="cb1-17"><a href="#cb1-17"></a>    <span class="kw">install.packages</span>(<span class="st">&quot;OneR&quot;</span>, <span class="dt">type =</span> <span class="st">&quot;binary&quot;</span>) </span>
<span id="cb1-18"><a href="#cb1-18"></a>    </span>
<span id="cb1-19"><a href="#cb1-19"></a>    <span class="co">#https://cran.r-project.org/web/packages/kableExtra/</span></span>
<span id="cb1-20"><a href="#cb1-20"></a>    <span class="kw">install.packages</span>(<span class="st">&#39;kableExtra&#39;</span>,  <span class="dt">type =</span> <span class="st">&quot;binary&quot;</span>)</span>
<span id="cb1-21"><a href="#cb1-21"></a>    </span>
<span id="cb1-22"><a href="#cb1-22"></a></span>
<span id="cb1-23"><a href="#cb1-23"></a><span class="co"># 2 A. You can  install package wizbionet from the GitHub use this command:</span></span>
<span id="cb1-24"><a href="#cb1-24"></a>    <span class="kw">install.packages</span>(<span class="st">&quot;remotes&quot;</span>)</span>
<span id="cb1-25"><a href="#cb1-25"></a>    remotes<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;wizbionet/wizbionet&quot;</span>, <span class="dt">dependencies =</span><span class="st">&quot;Depends&quot;</span>)</span>
<span id="cb1-26"><a href="#cb1-26"></a>    </span>
<span id="cb1-27"><a href="#cb1-27"></a><span class="co"># 2 B.Or install wizbionet from the wizbionet_0.99.0.tar.gz file downloaded from this link:</span></span>
<span id="cb1-28"><a href="#cb1-28"></a>    <span class="co">#https://github.com/wizbionet/wizbionet/raw/master/wizbionet_0.99.0.tar.gz</span></span>
<span id="cb1-29"><a href="#cb1-29"></a>    </span>
<span id="cb1-30"><a href="#cb1-30"></a>    <span class="kw">install.packages</span>(<span class="st">&quot;https://github.com/wizbionet/wizbionet/raw/master/wizbionet_0.99.0.tar.gz&quot;</span>, </span>
<span id="cb1-31"><a href="#cb1-31"></a>                 <span class="dt">repos =</span> <span class="ot">NULL</span>, <span class="dt">type =</span> <span class="st">&quot;source&quot;</span>, </span>
<span id="cb1-32"><a href="#cb1-32"></a>                 <span class="dt">dependencies =</span><span class="ot">TRUE</span></span>
<span id="cb1-33"><a href="#cb1-33"></a>                 )</span>
<span id="cb1-34"><a href="#cb1-34"></a></span>
<span id="cb1-35"><a href="#cb1-35"></a><span class="co">#Voila, load the package:</span></span>
<span id="cb1-36"><a href="#cb1-36"></a><span class="kw">library</span>(wizbionet)</span></code></pre></div>
</div>
<div id="wizbionet-workflow-example" class="section level2">
<h2>wizbionet workflow example</h2>
<div id="importing-gene-lists-to-the-r" class="section level3">
<h3>1. importing gene lists to the R</h3>
<p>This set of functions enables combining gene lists associated with your process of interest. You can copy columns from excel and paste them to R using command:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a><span class="co"># genelistGWAS&lt;- readClipboard()</span></span>
<span id="cb2-2"><a href="#cb2-2"></a><span class="co"># genelistGO&lt;- readClipboard()</span></span>
<span id="cb2-3"><a href="#cb2-3"></a><span class="co"># genelistKEGG&lt;- readClipboard()</span></span></code></pre></div>
</div>
<div id="updating-gene-symbols-annotation-using-ncbi_synonyms" class="section level3">
<h3>2. Updating gene symbols annotation using NCBI_synonyms()</h3>
<p>The common problem while working with different gene lists are not matching gene symbols which can generates duplicates while merging them. this package offers tool for efficient annotation of human genes according to NCBI nomenclature using function <strong>NCBI_synonyms()</strong>. Notice that gene EP3 has now symbol PTGER3, and SNT3 has symbol SNTB2.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>genelistGWAS&lt;-<span class="st"> </span><span class="kw">as.data.frame</span>(genelistGWAS)</span>
<span id="cb3-2"><a href="#cb3-2"></a><span class="kw">NCBI_synonyms</span>(genelistGWAS, <span class="st">&quot;genelistGWAS&quot;</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:left;">
EntrezID
</th>
<th style="text-align:left;">
ENSG_ID
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;color: red !important;">
SGCD
</td>
<td style="text-align:left;">
6444
</td>
<td style="text-align:left;">
ENSG00000170624
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;color: red !important;">
TBC1D8B
</td>
<td style="text-align:left;">
54885
</td>
<td style="text-align:left;">
ENSG00000133138
</td>
</tr>
<tr>
<td style="text-align:left;">
EP3
</td>
<td style="text-align:left;color: red !important;">
PTGER3
</td>
<td style="text-align:left;">
5733
</td>
<td style="text-align:left;">
ENSG00000050628
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;color: red !important;">
ANO5
</td>
<td style="text-align:left;">
203859
</td>
<td style="text-align:left;">
ENSG00000171714
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;color: red !important;">
FGFR1
</td>
<td style="text-align:left;">
2260
</td>
<td style="text-align:left;">
ENSG00000077782
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;color: red !important;">
PTGFR
</td>
<td style="text-align:left;">
5737
</td>
<td style="text-align:left;">
ENSG00000122420
</td>
</tr>
<tr>
<td style="text-align:left;">
SNT3
</td>
<td style="text-align:left;color: red !important;">
SNTB2
</td>
<td style="text-align:left;">
6645
</td>
<td style="text-align:left;">
ENSG00000168807
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;color: red !important;">
USP9Y
</td>
<td style="text-align:left;">
8287
</td>
<td style="text-align:left;">
ENSG00000114374
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;color: red !important;">
PTPRM
</td>
<td style="text-align:left;">
5797
</td>
<td style="text-align:left;">
ENSG00000173482
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;color: red !important;">
CRIM1
</td>
<td style="text-align:left;">
51232
</td>
<td style="text-align:left;">
ENSG00000150938
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;color: red !important;">
BRD1
</td>
<td style="text-align:left;">
23774
</td>
<td style="text-align:left;">
ENSG00000100425
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;color: red !important;">
RAD23B
</td>
<td style="text-align:left;">
5887
</td>
<td style="text-align:left;">
ENSG00000119318
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;color: red !important;">
PRKDC
</td>
<td style="text-align:left;">
5591
</td>
<td style="text-align:left;">
ENSG00000253729
</td>
</tr>
</tbody>
</table>
</div>
<div id="selecting-columns-based-on-syntax-using-col_selector" class="section level3">
<h3>3. Selecting columns based on syntax using col_selector()</h3>
<p>If you are not interested in other columns except for the official gene symbols you can use syntax specific function:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="kw">col_selector</span>(genelistGWAS, <span class="dt">key_words =</span> <span class="st">&quot;SymbolNCBI&quot;</span>)</span>
<span id="cb4-2"><a href="#cb4-2"></a><span class="co">#Use Regular Expression Syntax (https://rstudio.com/wp-content/uploads/2016/09/RegExCheatsheet.pdf) to precisely select columns:</span></span>
<span id="cb4-3"><a href="#cb4-3"></a><span class="co">#Meta characters . * + </span></span>
<span id="cb4-4"><a href="#cb4-4"></a><span class="co">#Anchors: ^ Start of the string; $ End of the string </span></span></code></pre></div>
</div>
<div id="binding-multiple-gene-lists-andor-data.frames-of-unequal-lenght-using-cbind_filler" class="section level3">
<h3>4. Binding multiple gene lists and/or data.frames of unequal lenght using cbind_filler()</h3>
<p>To be able to bind multiple gene lists and/or data.frames of unequal lenght u can use <strong>cbind_filler()</strong></p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a>output&lt;-<span class="st"> </span><span class="kw">cbind_filler</span>(<span class="kw">list</span>(genelistGWAS,genelistGO,genelistKEEG))</span></code></pre></div>
<p>If you are working with vectors after binding them you will need to add names manually. If the gene lists were data frames with headings you don’t need to do this.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a><span class="kw">names</span>(output)&lt;-<span class="kw">c</span>(<span class="st">&#39;genelistGWAS&#39;</span>,<span class="st">&#39;genelistGO&#39;</span>,<span class="st">&#39;genelistKEEG&#39;</span>)</span></code></pre></div>
<p>Below is example data frame merged using <strong>cbind_filler()</strong> function with suggestions of the sources of gene lists. The advantage of this function is that it allows to combine vectors/data frames of different lenght</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a>output&lt;-<span class="st"> </span><span class="kw">cbind_filler</span>(output)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
genelistGO
</th>
<th style="text-align:left;">
genelistKEGG
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
SGCD
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
TBC1D8B
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
NEBL
</td>
<td style="text-align:left;">
PTGER3
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
ANO5
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
FGFR1
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
PTGFR
</td>
</tr>
<tr>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
NELL1
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
NR2C2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
TGFB3
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
BRD1
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
RAD23B
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
PRKDC
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
IGSF10
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
AKIRIN1
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
HACD4
</td>
</tr>
<tr>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
C4orf32
</td>
</tr>
</tbody>
</table>
</div>
<div id="combining-and-summarizing-multiple-gene-lists-using-lists.combiner" class="section level3">
<h3>5. Combining and summarizing multiple gene lists using lists.combiner()</h3>
<p>When you have already a data frame build using <strong>cbind_filler()</strong> or just imported form txt/excel or csv file, we suggest to use function <strong>lists.combiner()</strong>. This function will merge all of those data frames and will provide information in which list a given gene was present and in how many lists in total it appeared.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>output&lt;-<span class="st"> </span><span class="kw">lists.combiner</span>(output)</span>
<span id="cb8-2"><a href="#cb8-2"></a><span class="kw">head</span>(output, <span class="dt">n=</span><span class="dv">15</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
input
</th>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
genelistGO
</th>
<th style="text-align:left;">
genelistKEGG
</th>
<th style="text-align:right;">
count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;color: red !important;">
1
</td>
</tr>
</tbody>
</table>
<p>Now you have a complete list of genes of interest (column input) and can easily see that gene PTGER3 was present in all of those data sets. You can also use function <strong>NCBI_synonyms()</strong> to add gene IDs.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a>output&lt;-<span class="st"> </span><span class="kw">NCBI_synonyms</span>(output, <span class="st">&quot;input&quot;</span>)</span>
<span id="cb9-2"><a href="#cb9-2"></a><span class="kw">head</span>(output, <span class="dt">n=</span><span class="dv">10</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
input
</th>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
genelistGO
</th>
<th style="text-align:left;">
genelistKEGG
</th>
<th style="text-align:right;">
count
</th>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:left;">
EntrezID
</th>
<th style="text-align:left;">
ENSG_ID
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;color: red !important;">
PTGER3
</td>
<td style="text-align:left;color: red !important;">
5733
</td>
<td style="text-align:left;color: red !important;">
ENSG00000050628
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;color: red !important;">
BRD1
</td>
<td style="text-align:left;color: red !important;">
23774
</td>
<td style="text-align:left;color: red !important;">
ENSG00000100425
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;color: red !important;">
RAD23B
</td>
<td style="text-align:left;color: red !important;">
5887
</td>
<td style="text-align:left;color: red !important;">
ENSG00000119318
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;color: red !important;">
PRKDC
</td>
<td style="text-align:left;color: red !important;">
5591
</td>
<td style="text-align:left;color: red !important;">
ENSG00000253729
</td>
</tr>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
SGCD
</td>
<td style="text-align:left;color: red !important;">
6444
</td>
<td style="text-align:left;color: red !important;">
ENSG00000170624
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
TBC1D8B
</td>
<td style="text-align:left;color: red !important;">
54885
</td>
<td style="text-align:left;color: red !important;">
ENSG00000133138
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
ANO5
</td>
<td style="text-align:left;color: red !important;">
203859
</td>
<td style="text-align:left;color: red !important;">
ENSG00000171714
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
FGFR1
</td>
<td style="text-align:left;color: red !important;">
2260
</td>
<td style="text-align:left;color: red !important;">
ENSG00000077782
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
PTGFR
</td>
<td style="text-align:left;color: red !important;">
5737
</td>
<td style="text-align:left;color: red !important;">
ENSG00000122420
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;color: red !important;">
PTPRM
</td>
<td style="text-align:left;color: red !important;">
5797
</td>
<td style="text-align:left;color: red !important;">
ENSG00000173482
</td>
</tr>
</tbody>
</table>
</div>
<div id="clusterization-of-gene-lists-using-clusterizer_oner" class="section level3">
<h3>6. Clusterization of gene lists using clusterizer_oneR()</h3>
<p>If you want to know genes associated with highest number of genes lists you can use function <strong>clusterizer_oneR()</strong>, which will divide genes from the “input” column into four clusters based on column “count”. clusterizer_oneR() utilzes Jenks natural breaks optimization algorithm from the original OneR package as a non-arbitrary classification dividing numbers of regulated genes into four categories (clusters). advantage of this method is non-arbitrary selection of top hits. You can cluster multiple numeric columns using this function. If you want to change the number of bins or method of the clusterization you can use function OneR::bin</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a>landmark_col,=<span class="st">&quot;SymbolNCBI&quot;</span></span>
<span id="cb10-2"><a href="#cb10-2"></a>cols_to_cluster=<span class="st">&quot;count&quot;</span></span>
<span id="cb10-3"><a href="#cb10-3"></a>output&lt;-<span class="st"> </span><span class="kw">clusterizer_oneR</span>(output, landmark_col, cols_to_cluster) </span>
<span id="cb10-4"><a href="#cb10-4"></a><span class="kw">head</span>(output, <span class="dt">n=</span><span class="dv">10</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
input
</th>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
genelistGO
</th>
<th style="text-align:left;">
genelistKEGG
</th>
<th style="text-align:right;">
count
</th>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:left;">
EntrezID
</th>
<th style="text-align:left;">
ENSG_ID
</th>
<th style="text-align:left;">
clus_count
</th>
<th style="text-align:left;">
clusNR_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
5733
</td>
<td style="text-align:left;">
ENSG00000050628
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
23774
</td>
<td style="text-align:left;">
ENSG00000100425
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
5887
</td>
<td style="text-align:left;">
ENSG00000119318
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
5591
</td>
<td style="text-align:left;">
ENSG00000253729
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
6444
</td>
<td style="text-align:left;">
ENSG00000170624
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
54885
</td>
<td style="text-align:left;">
ENSG00000133138
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
203859
</td>
<td style="text-align:left;">
ENSG00000171714
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
2260
</td>
<td style="text-align:left;">
ENSG00000077782
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
5737
</td>
<td style="text-align:left;">
ENSG00000122420
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
5797
</td>
<td style="text-align:left;">
ENSG00000173482
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
51232
</td>
<td style="text-align:left;">
ENSG00000150938
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
6645
</td>
<td style="text-align:left;">
ENSG00000168807
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
8287
</td>
<td style="text-align:left;">
ENSG00000114374
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
64403
</td>
<td style="text-align:left;">
ENSG00000139880
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
9480
</td>
<td style="text-align:left;">
ENSG00000119547
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
</tr>
</tbody>
</table>
</div>
<div id="clusterization-of-gene-lists-using-top_percent" class="section level3">
<h3>7. Clusterization of gene lists using top_percent()</h3>
<p>top_percent() is an alternative for <strong>clusterizer_oneR()</strong> which doesn’t deal well with continuous numbers like p-values or fold changes. This function enables comparison of data sets of different length. It is suggested to use it on gene lists which have associated numeric values. Prioritization of the analyzed gene lists can be based on the scores assigned after data aggregation and counting. This function helps to avoid arbitrary selection of top candidates, selecting top percent of genes for a given cutoff. It includes all genes close to a cutoff if they have same value. It generates new column with TRUE or FALSE value giving information if our gene was present in the top percents.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1"></a>landmark_col,=<span class="st">&quot;SymbolNCBI&quot;</span></span>
<span id="cb11-2"><a href="#cb11-2"></a>cols_to_cluster=<span class="st">&quot;count&quot;</span></span>
<span id="cb11-3"><a href="#cb11-3"></a>cutoff=<span class="dv">25</span></span>
<span id="cb11-4"><a href="#cb11-4"></a>output&lt;-<span class="st"> </span><span class="kw">clusterizer_oneR</span>(output, landmark_col, cols_to_cluster, cutoff) </span>
<span id="cb11-5"><a href="#cb11-5"></a><span class="kw">head</span>(output, <span class="dt">n=</span><span class="dv">10</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
input
</th>
<th style="text-align:left;">
genelistGWAS
</th>
<th style="text-align:left;">
genelistGO
</th>
<th style="text-align:left;">
genelistKEGG
</th>
<th style="text-align:right;">
count
</th>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:left;">
EntrezID
</th>
<th style="text-align:left;">
ENSG_ID
</th>
<th style="text-align:left;">
clus25p_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
5733
</td>
<td style="text-align:left;">
ENSG00000050628
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:left;">
23774
</td>
<td style="text-align:left;">
ENSG00000100425
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:left;">
5887
</td>
<td style="text-align:left;">
ENSG00000119318
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
5591
</td>
<td style="text-align:left;">
ENSG00000253729
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:left;">
6444
</td>
<td style="text-align:left;">
ENSG00000170624
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:left;">
54885
</td>
<td style="text-align:left;">
ENSG00000133138
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:left;">
203859
</td>
<td style="text-align:left;">
ENSG00000171714
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
2260
</td>
<td style="text-align:left;">
ENSG00000077782
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:left;">
5737
</td>
<td style="text-align:left;">
ENSG00000122420
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:left;">
5797
</td>
<td style="text-align:left;">
ENSG00000173482
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:left;">
51232
</td>
<td style="text-align:left;">
ENSG00000150938
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:left;">
6645
</td>
<td style="text-align:left;">
ENSG00000168807
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
8287
</td>
<td style="text-align:left;">
ENSG00000114374
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
64403
</td>
<td style="text-align:left;">
ENSG00000139880
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
9480
</td>
<td style="text-align:left;">
ENSG00000119547
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
</tbody>
</table>
</div>
<div id="merging-of-the-multiple-data-frames-with-headings-preservation-using-dataframe.merger" class="section level3">
<h3>8. Merging of the multiple data frames with headings preservation using dataframe.merger()</h3>
<p>As you can see in the output appeared two columns ‘clus_count’ and ‘clusNR_count’. First column provides logical information if gene was present in the top two clusters (cl1, cl2). Second column provides information in which cluster appeared this gene. You can combine this data frame with your another data frame prioritized in same way using common column (in this case “SymbolNCBI”) and function <strong>dataframe.merger()</strong>.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a>dataframe_list=<span class="kw">list</span>(<span class="st">&#39;output&#39;</span>=output, <span class="st">&#39;other_dataframe&#39;</span>=other_dataframe)</span>
<span id="cb12-2"><a href="#cb12-2"></a>ID_column=<span class="st">&quot;input&quot;</span></span>
<span id="cb12-3"><a href="#cb12-3"></a><span class="kw">dataframe.merger</span>(dataframe_list,  <span class="dt">ID_column=</span><span class="st">&quot;input&quot;</span>)</span></code></pre></div>
<p>If you are analyzing two data sets using this approach you can select genes present in both of them, so called top genes. This approach enables quick selection of top candidates across multiple datasates.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1"></a>landmark_col=<span class="st">&quot;SymbolNCBI&quot;</span> </span>
<span id="cb13-2"><a href="#cb13-2"></a>cols_to_cluster=<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;count&quot;</span>, <span class="st">&#39;count_regulatory_miRNA&#39;</span>)</span>
<span id="cb13-3"><a href="#cb13-3"></a>output&lt;-<span class="st"> </span><span class="kw">clusterizer_oneR</span>(output, landmark_col, cols_to_cluster) </span>
<span id="cb13-4"><a href="#cb13-4"></a><span class="kw">head</span>(output, <span class="dt">n=</span><span class="dv">10</span>)</span></code></pre></div>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:right;">
count
</th>
<th style="text-align:right;">
count_regulatory_miRNA
</th>
<th style="text-align:left;">
clus_count
</th>
<th style="text-align:left;">
clusNR_count
</th>
<th style="text-align:left;">
clus_count_regulatory_miRNA
</th>
<th style="text-align:left;">
clusNR_count_regulatory_miRNA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
BRD1
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
RAD23B
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
107
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
SGCD
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
44
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
TBC1D8B
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
56
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
46
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
63
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
54
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
CRIM1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
SNTB2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
26
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl4
</td>
</tr>
<tr>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
47
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
</tr>
<tr>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
88
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
</tbody>
</table>
</div>
<div id="aggregation-and-summarizing-of-the-rows-across-multiple-columns-using-col_agrecounter" class="section level3">
<h3>9. Aggregation and summarizing of the rows across multiple columns using col_agrecounter()</h3>
<p>Function <strong><em>col_agrecounter</em>()</strong>* allows to aggregates and summarize complex data frames with preserving names of the columns and providing: - aggregated occurrences for example genes or miRNAs in a column with preserved name and suffix &quot;_coll&quot; - count of occurrences from column &quot;_coll&quot; as collumn with suffix &quot;_count&quot;. After this function we suggest runing top_percent() or clusterizer_oneR() to see which miRNAs are the most interesting ones based on the number of regulated targets.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="co">#Example</span></span>
<span id="cb14-2"><a href="#cb14-2"></a>dat1&lt;-<span class="st"> </span><span class="kw">data.frame</span>(</span>
<span id="cb14-3"><a href="#cb14-3"></a>  <span class="dt">mature_miRNA=</span><span class="kw">c</span>(<span class="st">&#39;hsa-miR-195-5p&#39;</span>, <span class="st">&#39;hsa-miR-195-3p&#39;</span>,<span class="st">&#39;hsa-miR-195-5p&#39;</span>, <span class="st">&#39;hsa-miR-195-5p&#39;</span>,</span>
<span id="cb14-4"><a href="#cb14-4"></a>                 <span class="st">&#39;hsa-miR-4753-5p&#39;</span>, <span class="st">&#39;hsa-miR-4753-3p&#39;</span>),</span>
<span id="cb14-5"><a href="#cb14-5"></a>  <span class="dt">pre_miRNA=</span><span class="kw">c</span>(<span class="st">&#39;hsa-miR-195&#39;</span>, <span class="st">&#39;hsa-miR-195&#39;</span>,<span class="st">&#39;hsa-miR-195&#39;</span>, <span class="st">&#39;hsa-miR-195&#39;</span>,</span>
<span id="cb14-6"><a href="#cb14-6"></a>              <span class="st">&#39;hsa-miR-4753&#39;</span>, <span class="st">&#39;hsa-miR-4753&#39;</span>),</span>
<span id="cb14-7"><a href="#cb14-7"></a>  <span class="dt">Target=</span><span class="kw">c</span>(<span class="st">&#39;CDH24&#39;</span>, <span class="st">&#39;PAX1&#39;</span>, <span class="st">&#39;PTGER3&#39;</span>,   <span class="st">&#39;ONECUT2&#39;</span>,  <span class="st">&#39;TGFB3&#39;</span>,    <span class="st">&#39;FGFR1&#39;</span>))</span>
<span id="cb14-8"><a href="#cb14-8"></a></span>
<span id="cb14-9"><a href="#cb14-9"></a></span>
<span id="cb14-10"><a href="#cb14-10"></a><span class="co">#set parameters####</span></span>
<span id="cb14-11"><a href="#cb14-11"></a></span>
<span id="cb14-12"><a href="#cb14-12"></a><span class="co">#dataframe for aggegation and counting (A&amp;C)</span></span>
<span id="cb14-13"><a href="#cb14-13"></a>    inputDF&lt;-dat1 </span>
<span id="cb14-14"><a href="#cb14-14"></a><span class="co">#selected column names which will be A&amp;C</span></span>
<span id="cb14-15"><a href="#cb14-15"></a>    col_names &lt;-<span class="st"> </span><span class="kw">c</span>( <span class="st">&quot;Target&quot;</span>)</span>
<span id="cb14-16"><a href="#cb14-16"></a><span class="co">#landmark column for A&amp;C columns,other columns will be aggregated based on this</span></span>
<span id="cb14-17"><a href="#cb14-17"></a>    col_collapse &lt;-<span class="st"> &quot;pre_miRNA&quot;</span>       </span>
<span id="cb14-18"><a href="#cb14-18"></a><span class="co">#vector of content of the landmark column &quot;col_collapse&quot; as vector. </span></span>
<span id="cb14-19"><a href="#cb14-19"></a><span class="co">#You can use internal col_to_string() function</span></span>
<span id="cb14-20"><a href="#cb14-20"></a>    </span>
<span id="cb14-21"><a href="#cb14-21"></a>    rows_collapse &lt;-<span class="kw">col_to_string</span>(inputDF<span class="op">$</span>pre_miRNA)</span>
<span id="cb14-22"><a href="#cb14-22"></a><span class="co">#additional landmark column which will not allow to deduplicate. useful when you want to analyze pre-miRNAs instead of mature mirNAs</span></span>
<span id="cb14-23"><a href="#cb14-23"></a>    control_col&lt;-<span class="st"> &quot;mature_miRNA&quot;</span></span>
<span id="cb14-24"><a href="#cb14-24"></a></span>
<span id="cb14-25"><a href="#cb14-25"></a><span class="co">#run function    </span></span>
<span id="cb14-26"><a href="#cb14-26"></a>output&lt;-<span class="st"> </span><span class="kw">col_agrecounter</span>(inputDF, col_names, col_collapse , rows_collapse, control_col)</span></code></pre></div>
<pre><code>#&gt; [1] &quot;inputDF- dataframe for aggegation and counting (A&amp;C)&quot;
#&gt; [1] &quot;col_names- column names which will be A&amp;C&quot;
#&gt; [1] &quot;col_collapse- landmark column for A&amp;C columns&quot;
#&gt; [1] &quot;rows_collapse- content of the column  landmark column all_of(col_collapse) as vector &quot;
#&gt; [1] &quot;control_col- additional landmark column&quot;
#&gt; [1] &quot;hsa-miR-195 count: 4 Target&quot;
#&gt; [1] &quot;hsa-miR-4753 count: 2 Target&quot;</code></pre>
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
mature_miRNA
</th>
<th style="text-align:left;">
pre_miRNA
</th>
<th style="text-align:left;">
Target
</th>
<th style="text-align:left;">
control
</th>
<th style="text-align:left;">
Target_coll
</th>
<th style="text-align:right;">
Target_COUNT
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
hsa-miR-195-3p
</td>
<td style="text-align:left;">
hsa-miR-195
</td>
<td style="text-align:left;">
PAX1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
CDH24|ONECUT2|PAX1|PTGER3
</td>
<td style="text-align:right;color: red !important;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
hsa-miR-195-5p
</td>
<td style="text-align:left;">
hsa-miR-195
</td>
<td style="text-align:left;">
CDH24
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
CDH24|ONECUT2|PAX1|PTGER3
</td>
<td style="text-align:right;color: red !important;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
hsa-miR-195-5p
</td>
<td style="text-align:left;">
hsa-miR-195
</td>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
CDH24|ONECUT2|PAX1|PTGER3
</td>
<td style="text-align:right;color: red !important;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
hsa-miR-195-5p
</td>
<td style="text-align:left;">
hsa-miR-195
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
CDH24|ONECUT2|PAX1|PTGER3
</td>
<td style="text-align:right;color: red !important;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
hsa-miR-4753-3p
</td>
<td style="text-align:left;">
hsa-miR-4753
</td>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
FGFR1|TGFB3
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
hsa-miR-4753-5p
</td>
<td style="text-align:left;">
hsa-miR-4753
</td>
<td style="text-align:left;">
TGFB3
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
FGFR1|TGFB3
</td>
<td style="text-align:right;color: red !important;">
2
</td>
</tr>
</tbody>
</table>
</div>
<div id="identification-of-the-top-genes-across-datasets" class="section level3">
<h3>10. Identification of the top genes across datasets</h3>
<p>Using base::subset function you can extract top genes which got high scores in both columns in this case “count” and ‘count_regulatory_miRNA’.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>output&lt;-<span class="kw">subset</span>(output, output<span class="op">$</span>clus_count<span class="op">==</span><span class="ot">TRUE</span> <span class="op">&amp;</span><span class="st"> </span>output<span class="op">$</span>clus_count_regulatory_miRNA<span class="op">==</span><span class="ot">TRUE</span> ) </span></code></pre></div>
Example data frame:
<table class="table table-condensed" style="font-size: 9px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;">
SymbolNCBI
</th>
<th style="text-align:right;">
count
</th>
<th style="text-align:right;">
count_regulatory_miRNA
</th>
<th style="text-align:left;">
clus_count
</th>
<th style="text-align:left;">
clusNR_count
</th>
<th style="text-align:left;">
clus_count_regulatory_miRNA
</th>
<th style="text-align:left;">
clusNR_count_regulatory_miRNA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
PTGER3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
69
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO5
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
162
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
</tr>
<tr>
<td style="text-align:left;">
FGFR1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTGFR
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
82
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
<tr>
<td style="text-align:left;">
PTPRM
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
</tr>
</tbody>
</table>
</div>
<div id="identification-of-the-top-mirnas-and-top-targets-using-topmirna_toptarget" class="section level3">
<h3>11. Identification of the top miRNAs and top targets using topmiRNA_toptarget()</h3>
<p>As top miRNAs we define miRNAs associated with your process of interest and regulating highest number of deferentially expressed genes (DE genes). If you have expression data for miRNAs and mRNAs you can perform target predictions to identify pairs of oppositely regulated miRNAs and genes use function <strong>topmiRNA_toptarget()</strong>. This function retrieves miRNA-target interactions and identify miRNAs and genes with highest number of analyzed interactions with miRNAs. You can also use only targets associated with a specific process to identify top miRNAs regulating it.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1"></a><span class="co">#generate and modify list of arguments. Important: Don&#39;t add mirna=  and  target= fields they are already included as DEmir_up,DEgenes_down,DEmir_down, DEgenes_up!</span></span>
<span id="cb17-2"><a href="#cb17-2"></a>multimir_args&lt;-<span class="st"> </span><span class="kw">as.list</span>(<span class="kw">args</span>(multiMiR<span class="op">::</span>get_multimir)) </span>
<span id="cb17-3"><a href="#cb17-3"></a></span>
<span id="cb17-4"><a href="#cb17-4"></a><span class="co">#add lists of DE miRNAs and genes</span></span>
<span id="cb17-5"><a href="#cb17-5"></a>DEmir_up&lt;-<span class="kw">c</span>(<span class="st">&#39;hsa-miR-150-5p&#39;</span>,<span class="st">&#39;hsa-miR-448-5p&#39;</span>,<span class="st">&#39;hsa-miR-448-3p&#39;</span>,</span>
<span id="cb17-6"><a href="#cb17-6"></a>            <span class="st">&#39;hsa-miR-493-5p&#39;</span>,<span class="st">&#39;hsa-miR-493-3p&#39;</span>) <span class="co"># example DE miRNAs</span></span>
<span id="cb17-7"><a href="#cb17-7"></a>DEgenes_down&lt;-<span class="kw">c</span>(<span class="st">&#39;5797&#39;</span>,<span class="st">&#39;8826&#39;</span>,<span class="st">&#39;7994&#39;</span>,<span class="st">&#39;2775&#39;</span>,<span class="st">&#39;7182&#39;</span>,<span class="st">&#39;79647&#39;</span>,<span class="st">&#39;5733&#39;</span>,</span>
<span id="cb17-8"><a href="#cb17-8"></a>                <span class="st">&#39;158158&#39;</span>,<span class="st">&#39;9480&#39;</span>,<span class="st">&#39;8626&#39;</span>,<span class="st">&#39;50636&#39;</span>) <span class="co"># example DE genes</span></span>
<span id="cb17-9"><a href="#cb17-9"></a>DEmir_down&lt;-<span class="kw">c</span>(<span class="st">&#39;hsa-miR-4731-5p&#39;</span>,<span class="st">&#39;hsa-miR-541-3p&#39;</span>,<span class="st">&#39;hsa-miR-449b-5p&#39;</span>,<span class="st">&#39;hsa-miR-541-5p&#39;</span>)</span>
<span id="cb17-10"><a href="#cb17-10"></a>DEgenes_up&lt;-<span class="kw">c</span>(<span class="st">&#39;203859&#39;</span>,<span class="st">&#39;4745&#39;</span>,<span class="st">&#39;4916&#39;</span>,<span class="st">&#39;126298&#39;</span>,<span class="st">&#39;2258&#39;</span>,<span class="st">&#39;8464&#39;</span>,<span class="st">&#39;55917&#39;</span>,<span class="st">&#39;23450&#39;</span>,<span class="st">&#39;29767&#39;</span>)</span>
<span id="cb17-11"><a href="#cb17-11"></a>mirna_type&lt;-<span class="st">&quot;pre_mir&quot;</span> <span class="co"># &quot;mature_mir&quot;</span></span>
<span id="cb17-12"><a href="#cb17-12"></a>multimir_args=<span class="st"> </span><span class="kw">list</span>(<span class="dt">url =</span> <span class="ot">NULL</span>,</span>
<span id="cb17-13"><a href="#cb17-13"></a>                    <span class="dt">org =</span> <span class="st">&quot;hsa&quot;</span>,</span>
<span id="cb17-14"><a href="#cb17-14"></a>                    <span class="dt">table =</span> <span class="st">&quot;all&quot;</span>,</span>
<span id="cb17-15"><a href="#cb17-15"></a>                    <span class="dt">predicted.cutoff =</span> <span class="dv">10</span>,</span>
<span id="cb17-16"><a href="#cb17-16"></a>                    <span class="dt">predicted.cutoff.type =</span> <span class="st">&quot;p&quot;</span>,</span>
<span id="cb17-17"><a href="#cb17-17"></a>                    <span class="dt">predicted.site =</span> <span class="st">&quot;conserved&quot;</span></span>
<span id="cb17-18"><a href="#cb17-18"></a>)</span>
<span id="cb17-19"><a href="#cb17-19"></a></span>
<span id="cb17-20"><a href="#cb17-20"></a><span class="co">#execute function</span></span>
<span id="cb17-21"><a href="#cb17-21"></a>output&lt;-<span class="st"> </span><span class="kw">topmiRNA_toptarget</span>(DEmir_up,DEgenes_down,                                       DEmir_down, DEgenes_up, multimir_args,mirna_type)</span>
<span id="cb17-22"><a href="#cb17-22"></a></span>
<span id="cb17-23"><a href="#cb17-23"></a></span>
<span id="cb17-24"><a href="#cb17-24"></a><span class="co">#execute function</span></span>
<span id="cb17-25"><a href="#cb17-25"></a>output&lt;-wizbionet<span class="op">::</span><span class="kw">topmiRNA_toptarget</span>(DEmir_up,DEgenes_down,DEmir_down, </span>
<span id="cb17-26"><a href="#cb17-26"></a>                                      DEgenes_up, mir_type, multimir_args)</span>
<span id="cb17-27"><a href="#cb17-27"></a>output<span class="op">$</span>multimir_output</span>
<span id="cb17-28"><a href="#cb17-28"></a>output<span class="op">$</span>top_miR</span>
<span id="cb17-29"><a href="#cb17-29"></a>output<span class="op">$</span>top_gene</span></code></pre></div>
<div id="output-of-the-topmirna_toptarget-is-a-list-with-three-data-frames-named" class="section level4">
<h4>Output of the <strong>topmiRNA_toptarget()</strong> is a list() with three data frames named:</h4>
<div id="a.-multimir_output" class="section level5">
<h5><strong>A. multimir_output</strong></h5>
<p>Data frame <em>multimir_output</em> is based on original output from the multiMiR::get_multimir(), but integrates two separate analyses for up and down regulated miRNAs. Directionality of expression is shown in the column “multimir”. Additionally, at the end of the dataframe is added column “premir” where are present names of analyzed miRNAs before maturation.</p>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
database
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_acc
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_ensembl
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
experiment
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
support_type
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pubmed_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
type
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
score
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
disease_drug
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
paper_pubmedID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
multimir
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
premir
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mirtarbase
</td>
<td style="text-align:left;">
MIMAT0000451
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
ANO7
</td>
<td style="text-align:left;">
50636
</td>
<td style="text-align:left;">
ENSG00000146205
</td>
<td style="text-align:left;">
HITS-CLIP
</td>
<td style="text-align:left;">
Functional MTI (Weak)
</td>
<td style="text-align:left;">
23824327
</td>
<td style="text-align:left;">
validated
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150
</td>
</tr>
<tr>
<td style="text-align:left;">
mirtarbase
</td>
<td style="text-align:left;">
MIMAT0000451
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
AKIRIN1
</td>
<td style="text-align:left;">
79647
</td>
<td style="text-align:left;">
ENSG00000174574
</td>
<td style="text-align:left;">
HITS-CLIP
</td>
<td style="text-align:left;">
Functional MTI (Weak)
</td>
<td style="text-align:left;">
23824327
</td>
<td style="text-align:left;">
validated
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150
</td>
</tr>
<tr>
<td style="text-align:left;">
tarbase
</td>
<td style="text-align:left;">
MIMAT0000451
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
IQGAP1
</td>
<td style="text-align:left;">
8826
</td>
<td style="text-align:left;">
ENSG00000140575
</td>
<td style="text-align:left;">
Degradome sequencing
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
validated
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150
</td>
</tr>
<tr>
<td style="text-align:left;">
tarbase
</td>
<td style="text-align:left;">
MIMAT0003161
</td>
<td style="text-align:left;">
hsa-miR-493-3p
</td>
<td style="text-align:left;">
IQGAP1
</td>
<td style="text-align:left;">
8826
</td>
<td style="text-align:left;">
ENSG00000140575
</td>
<td style="text-align:left;">
Degradome sequencing
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
validated
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
diana_microt
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
7994
</td>
<td style="text-align:left;">
ENSG00000083168
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.998
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
diana_microt
</td>
<td style="text-align:left;">
MIMAT0000451
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
9480
</td>
<td style="text-align:left;">
ENSG00000119547
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.936
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150
</td>
</tr>
<tr>
<td style="text-align:left;">
diana_microt
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
TP63
</td>
<td style="text-align:left;">
8626
</td>
<td style="text-align:left;">
ENSG00000073282
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.88
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
elmmo
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
7994
</td>
<td style="text-align:left;">
ENSG00000083168
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.881
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
elmmo
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
9480
</td>
<td style="text-align:left;">
ENSG00000119547
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.742
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
elmmo
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
NR2C2
</td>
<td style="text-align:left;">
7182
</td>
<td style="text-align:left;">
ENSG00000177463
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.598
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
elmmo
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
TP63
</td>
<td style="text-align:left;">
8626
</td>
<td style="text-align:left;">
ENSG00000073282
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
0.532
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
microcosm
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
NR2C2
</td>
<td style="text-align:left;">
7182
</td>
<td style="text-align:left;">
ENSG00000177463
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
18.3062
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
microcosm
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
ANO7
</td>
<td style="text-align:left;">
50636
</td>
<td style="text-align:left;">
ENSG00000146205
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
17.8267
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
pictar
</td>
<td style="text-align:left;">
MIMAT0002813
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
7994
</td>
<td style="text-align:left;">
ENSG00000083168
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
29.652
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-493
</td>
</tr>
<tr>
<td style="text-align:left;">
pita
</td>
<td style="text-align:left;">
MIMAT0000451
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
7994
</td>
<td style="text-align:left;">
ENSG00000083168
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
predicted
</td>
<td style="text-align:left;">
-10.53
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;color: red !important;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="b.-top_mir" class="section level5">
<h5><strong>B. top_miR</strong></h5>
<p>Data frame <em>top_miR</em> has aggregated and prioritized results from get_multimir output showing number of genes associated with pre-miRNAs. It has columns with name <em>clus_…</em> providing logical information if gene was in top 2 clusters (cl1 and cl2) ~top 20 percents and column <em>clusNR_..</em>. providing information in which cluster the gene was present (cl1,cl2,cl3,cl4). It enables quick identification of the miRNAs regulating highest number of analyzed targets</p>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
premir
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_id_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_id_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
multimir_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
mature_mirna_acc
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_symbol_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_symbol_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
type_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
type_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clus_target_symbol_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clusNR_target_symbol_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clus25p_target_symbol_COUNT
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
hsa-miR-493
</td>
<td style="text-align:left;">
hsa-miR-493-3p
</td>
<td style="text-align:left;">
hsa-miR-493-3p|hsa-miR-493-5p
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
MIMAT0003161
</td>
<td style="text-align:left;color: red !important;">
ANO7|GNAO1|IQGAP1|KAT6A|NR2C2|ONECUT2|TP63
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-493
</td>
<td style="text-align:left;">
hsa-miR-493-5p
</td>
<td style="text-align:left;">
hsa-miR-493-3p|hsa-miR-493-5p
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
MIMAT0002813
</td>
<td style="text-align:left;color: red !important;">
ANO7|GNAO1|IQGAP1|KAT6A|NR2C2|ONECUT2|TP63
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-449b
</td>
<td style="text-align:left;">
hsa-miR-449b-5p
</td>
<td style="text-align:left;">
hsa-miR-449b-5p
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
data_DEmir_down_DEgenes_up
</td>
<td style="text-align:left;color: red !important;">
MIMAT0003327
</td>
<td style="text-align:left;color: red !important;">
CTTNBP2NL|IRGQ|NTRK3|SF3B3|SUPT3H
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-150
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
hsa-miR-150-5p
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
data_DEmir_up_DEgenes_down
</td>
<td style="text-align:left;color: red !important;">
MIMAT0000451
</td>
<td style="text-align:left;color: red !important;">
AKIRIN1|ANO7|IQGAP1|KAT6A|ONECUT2
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
TRUE
</td>
<td style="text-align:left;color: red !important;">
cl2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-541
</td>
<td style="text-align:left;">
hsa-miR-541-3p
</td>
<td style="text-align:left;">
hsa-miR-541-3p|hsa-miR-541-5p
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
data_DEmir_down_DEgenes_up
</td>
<td style="text-align:left;color: red !important;">
MIMAT0004920
</td>
<td style="text-align:left;color: red !important;">
NTRK3|SF3B3|TMOD2
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-541
</td>
<td style="text-align:left;">
hsa-miR-541-5p
</td>
<td style="text-align:left;">
hsa-miR-541-3p|hsa-miR-541-5p
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
data_DEmir_down_DEgenes_up
</td>
<td style="text-align:left;color: red !important;">
MIMAT0004919
</td>
<td style="text-align:left;color: red !important;">
NTRK3|SF3B3|TMOD2
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl3
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
hsa-miR-4731
</td>
<td style="text-align:left;">
hsa-miR-4731-5p
</td>
<td style="text-align:left;">
hsa-miR-4731-5p
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
data_DEmir_down_DEgenes_up
</td>
<td style="text-align:left;color: red !important;">
MIMAT0019853
</td>
<td style="text-align:left;color: red !important;">
IRGQ|SF3B3
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
predicted|validated
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
FALSE
</td>
<td style="text-align:left;color: red !important;">
cl4
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="c.-top_gene" class="section level5">
<h5><strong>C. top_gene</strong></h5>
<p>Data frame <em>top_gene</em> has aggregated and prioritized results from get_multimir function showing number of genes associated with analyzed targets. It also has columns with information if gene is in top 20%.</p>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_entrez
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_ensembl_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
premir_coll
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
premir_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clus_premir_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clusNR_premir_COUNT
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
clus25p_premir_COUNT
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
SF3B3
</td>
<td style="text-align:left;">
23450
</td>
<td style="text-align:left;">
ENSG00000189091
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-449b|hsa-miR-4731|hsa-miR-541
</td>
<td style="text-align:left;color: red !important;">
3
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl1
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
</tr>
<tr>
<td style="text-align:left;">
IRGQ
</td>
<td style="text-align:left;">
126298
</td>
<td style="text-align:left;">
ENSG00000167378
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-449b|hsa-miR-4731
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
NTRK3
</td>
<td style="text-align:left;">
4916
</td>
<td style="text-align:left;">
ENSG00000140538
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-449b|hsa-miR-541
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
ANO7
</td>
<td style="text-align:left;">
50636
</td>
<td style="text-align:left;">
ENSG00000146205
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150|hsa-miR-493
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
IQGAP1
</td>
<td style="text-align:left;">
8826
</td>
<td style="text-align:left;">
ENSG00000140575
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150|hsa-miR-493
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
KAT6A
</td>
<td style="text-align:left;">
7994
</td>
<td style="text-align:left;">
ENSG00000083168
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150|hsa-miR-493
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
9480
</td>
<td style="text-align:left;">
ENSG00000119547
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-150|hsa-miR-493
</td>
<td style="text-align:left;color: red !important;">
2
</td>
<td style="text-align:left;color: red !important;">
TRUE
</td>
<td style="text-align:left;">
cl2
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
TMOD2
</td>
<td style="text-align:left;">
29767
</td>
<td style="text-align:left;">
ENSG00000128872
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-541
</td>
<td style="text-align:left;color: red !important;">
1
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
SUPT3H
</td>
<td style="text-align:left;">
8464
</td>
<td style="text-align:left;">
ENSG00000196284
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-449b
</td>
<td style="text-align:left;color: red !important;">
1
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
<tr>
<td style="text-align:left;">
CTTNBP2NL
</td>
<td style="text-align:left;">
55917
</td>
<td style="text-align:left;">
ENSG00000143079
</td>
<td style="text-align:left;color: red !important;">
hsa-miR-449b
</td>
<td style="text-align:left;color: red !important;">
1
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
<td style="text-align:left;">
cl3
</td>
<td style="text-align:left;color: red !important;">
FALSE
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="identification-of-the-top-long-non-coding-rnas-and-their-targets-using-lnc_rnarna_scanner" class="section level3">
<h3>12. Identification of the top long-non coding RNAs and their targets using lnc_RNARNA_scanner()</h3>
<p>Function <strong><em>lnc_RNARNA_scanner</em></strong> identify targets of lncRNAs by performing automatized mining of the <a href="http://rtools.cbrc.jp/cgi-bin/RNARNA/index.pl">link</a> database using ENST transcript IDs (ENST_input) as an input. It enables identification of so-called top lncRNAs associated with your process of interest and regulating highest number of deferentially expressed genes (DE genes) <strong>ENST_input</strong> is a vector of characters (human ENST transcript numbers). RNARNAdb will show information during the analysis if non coding transcript was present in the database. Careful, it sometimes also recognizes miRNAs.</p>
<p><strong>analyze_onlyDEtargets</strong> TRUE or FALSE (logical), it says if you want to check all possible lncRNA targets or only DE targets.</p>
<p><strong>ENST_targets</strong> vector of characters of human ENST numbers of your interest, usually DE mRNAs</p>
<p><strong>nr_top_genes</strong> is a number number of top targets you want to select from all possible targets of lncRNAs. Default value for RNARNA database top targets output set by us is 500 if you want to compare it with DE genes. Your ENST_targets will be subtracted from this number.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a><span class="co">#Example of the analysis</span></span>
<span id="cb18-2"><a href="#cb18-2"></a>ENST_input&lt;-<span class="kw">c</span>(<span class="st">&#39;ENST00000436290&#39;</span>, <span class="st">&#39;ENST00000432892&#39;</span>, <span class="st">&#39;ENST00000454526&#39;</span>,</span>
<span id="cb18-3"><a href="#cb18-3"></a><span class="st">&#39;ENST00000423793&#39;</span>, <span class="st">&#39;ENST00000436960&#39;</span>,<span class="st">&#39;ENST00000412754&#39;</span></span>
<span id="cb18-4"><a href="#cb18-4"></a>)</span>
<span id="cb18-5"><a href="#cb18-5"></a></span>
<span id="cb18-6"><a href="#cb18-6"></a>ENST_targets&lt;-<span class="kw">c</span>(<span class="st">&#39;ENST00000491143&#39;</span>, <span class="st">&#39;ENST00000314191&#39;</span>,</span>
<span id="cb18-7"><a href="#cb18-7"></a><span class="st">&#39;ENST00000309733&#39;</span>, <span class="st">&#39;ENST00000377122&#39;</span>, <span class="st">&#39;ENST00000338981&#39;</span>,</span>
<span id="cb18-8"><a href="#cb18-8"></a><span class="st">&#39;ENST00000495827&#39;</span>, <span class="st">&#39;ENST00000425708&#39;</span>, <span class="st">&#39;ENST00000272748&#39;</span>,</span>
<span id="cb18-9"><a href="#cb18-9"></a>...)</span>
<span id="cb18-10"><a href="#cb18-10"></a>nr_top_genes=<span class="dv">500</span></span>
<span id="cb18-11"><a href="#cb18-11"></a>analyze_onlyDEtargets=<span class="ot">FALSE</span></span>
<span id="cb18-12"><a href="#cb18-12"></a>output&lt;-<span class="st"> </span><span class="kw">lnc_RNARNA_scanner</span>(ENST_input, analyze_onlyDEtargets,</span>
<span id="cb18-13"><a href="#cb18-13"></a>ENST_targets ,nr_top_genes)</span>
<span id="cb18-14"><a href="#cb18-14"></a><span class="kw">names</span>(output)</span></code></pre></div>
<div id="lnc_rnarna_scanner-output" class="section level4">
<h4>lnc_RNARNA_scanner output</h4>
<p>Output of <strong>lnc_RNARNA_scanner</strong> function is a list of data frames. To access the lists use $ sign or[[]]. All outputs have column &quot;“full_id” which describes interacting RNAs with your lncRNAs and provide a full name of the lncRNA for example: “List of RNAs interacting with ENST00000423793(RP11-432J22.2)”. You can use this column for aggregating and summarizing those datasets using <strong>col_agrecounter()</strong> and than <strong>clusterizer_oneR()</strong> to obtain top lncRNAs or top targets.</p>
<div id="a.-if-analyze_onlydetargets-was-set-as-true" class="section level5">
<h5><strong>A. if analyze_onlyDEtargets= was set as TRUE</strong></h5>
The output will include list <em>“lncRNA_DEtargets”</em> which has only top DE targets of analyzed lncRNAs
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Rank
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ensemble_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Position_hg19
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MinEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_query
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_target
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
location
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SumEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
nc_id
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
ENST00000238682
</td>
<td style="text-align:left;">
TGFB3
</td>
<td style="text-align:left;">
chr14:76424546-76447534
</td>
<td style="text-align:left;">
-39.4
</td>
<td style="text-align:left;">
1357-1406
</td>
<td style="text-align:left;">
1791-1840
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-579.4
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
259
</td>
<td style="text-align:left;">
ENST00000265354
</td>
<td style="text-align:left;">
SRF
</td>
<td style="text-align:left;">
chr6:43139038-43149243
</td>
<td style="text-align:left;">
-34.4
</td>
<td style="text-align:left;">
1357-1418
</td>
<td style="text-align:left;">
3068-3129
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-452.8
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
393
</td>
<td style="text-align:left;">
ENST00000392806
</td>
<td style="text-align:left;">
TMEM119
</td>
<td style="text-align:left;">
chr12:108983623-108991900
</td>
<td style="text-align:left;">
-32.7
</td>
<td style="text-align:left;">
1349-1403
</td>
<td style="text-align:left;">
764-818
</td>
<td style="text-align:left;">
CDS
</td>
<td style="text-align:left;">
-366.5
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
ENST00000491143
</td>
<td style="text-align:left;">
ONECUT2
</td>
<td style="text-align:left;">
chr18:55102918-55158529
</td>
<td style="text-align:left;">
-22.4
</td>
<td style="text-align:left;">
462-512
</td>
<td style="text-align:left;">
11203-11253
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-5288.2
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
249
</td>
<td style="text-align:left;">
ENST00000314191
</td>
<td style="text-align:left;">
PRKDC
</td>
<td style="text-align:left;">
chr8:48685670-48872743
</td>
<td style="text-align:left;">
-23.1
</td>
<td style="text-align:left;">
1353-1400
</td>
<td style="text-align:left;">
3325-3372
</td>
<td style="text-align:left;">
CDS
</td>
<td style="text-align:left;">
-3802.0
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
271
</td>
<td style="text-align:left;">
ENST00000495827
</td>
<td style="text-align:left;">
PTPLAD2
</td>
<td style="text-align:left;">
chr9:20999515-21031635
</td>
<td style="text-align:left;">
-23.6
</td>
<td style="text-align:left;">
601-649
</td>
<td style="text-align:left;">
3505-3553
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-3730.7
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
289
</td>
<td style="text-align:left;">
ENST00000338981
</td>
<td style="text-align:left;">
USP9Y
</td>
<td style="text-align:left;">
chrY:14813161-14972764
</td>
<td style="text-align:left;">
-20.8
</td>
<td style="text-align:left;">
528-578
</td>
<td style="text-align:left;">
6651-6701
</td>
<td style="text-align:left;">
CDS
</td>
<td style="text-align:left;">
-3680.9
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
295
</td>
<td style="text-align:left;">
ENST00000377122
</td>
<td style="text-align:left;">
NEBL
</td>
<td style="text-align:left;">
chr10:21068903-21186531
</td>
<td style="text-align:left;">
-26.3
</td>
<td style="text-align:left;">
872-927
</td>
<td style="text-align:left;">
1499-1554
</td>
<td style="text-align:left;">
CDS
</td>
<td style="text-align:left;">
-3638.4
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
305
</td>
<td style="text-align:left;">
ENST00000309733
</td>
<td style="text-align:left;">
C4orf32
</td>
<td style="text-align:left;">
chr4:113066554-113116412
</td>
<td style="text-align:left;">
-19.3
</td>
<td style="text-align:left;">
1425-1472
</td>
<td style="text-align:left;">
4944-4991
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-3609.7
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
428
</td>
<td style="text-align:left;">
ENST00000282466
</td>
<td style="text-align:left;">
IGSF10
</td>
<td style="text-align:left;">
chr3:151151283-151176497
</td>
<td style="text-align:left;">
-18.8
</td>
<td style="text-align:left;">
463-510
</td>
<td style="text-align:left;">
7622-7669
</td>
<td style="text-align:left;">
CDS
</td>
<td style="text-align:left;">
-3339.0
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
sumEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="b.-if-analyze_onlydetargets-was-set-as-false" class="section level5">
<h5><strong>B. if analyze_onlyDEtargets= was set as FALSE</strong></h5>
The output will include list <em>“lncRNA_AllTargets”</em> which has all possible top targets of analyzed lncRNAs
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Rank
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ensemble_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Position_hg19
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MinEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_query
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_target
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
location
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SumEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
target_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
nc_id
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
ENST00000607599
</td>
<td style="text-align:left;">
CHURC1
</td>
<td style="text-align:left;">
chr14:65381080-65402086
</td>
<td style="text-align:left;">
-60.6
</td>
<td style="text-align:left;">
775-836
</td>
<td style="text-align:left;">
1206-1267
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-1678.9
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
ENST00000373833
</td>
<td style="text-align:left;">
RCC1
</td>
<td style="text-align:left;">
chr1:28832570-28865812
</td>
<td style="text-align:left;">
-50.4
</td>
<td style="text-align:left;">
1354-1417
</td>
<td style="text-align:left;">
2332-2395
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-753.8
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
ENST00000599531
</td>
<td style="text-align:left;">
PNMAL2
</td>
<td style="text-align:left;">
chr19:46994449-46999755
</td>
<td style="text-align:left;">
-49.0
</td>
<td style="text-align:left;">
1353-1404
</td>
<td style="text-align:left;">
3518-3569
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-807.4
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
ENST00000329047
</td>
<td style="text-align:left;">
SEPT9
</td>
<td style="text-align:left;">
chr17:75315598-75496674
</td>
<td style="text-align:left;">
-46.0
</td>
<td style="text-align:left;">
1359-1408
</td>
<td style="text-align:left;">
701-750
</td>
<td style="text-align:left;">
UTR5
</td>
<td style="text-align:left;">
-762.7
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
ENST00000507938
</td>
<td style="text-align:left;">
AC090587.2
</td>
<td style="text-align:left;">
chr11:3837983-3843130
</td>
<td style="text-align:left;">
-45.3
</td>
<td style="text-align:left;">
1369-1416
</td>
<td style="text-align:left;">
846-893
</td>
<td style="text-align:left;">
ncRNA
</td>
<td style="text-align:left;">
-563.6
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
ENST00000382764
</td>
<td style="text-align:left;">
CYorf17
</td>
<td style="text-align:left;">
chrY:23544841-23548246
</td>
<td style="text-align:left;">
-44.9
</td>
<td style="text-align:left;">
1370-1419
</td>
<td style="text-align:left;">
690-739
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-240.1
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
ENST00000392857
</td>
<td style="text-align:left;">
ORC4
</td>
<td style="text-align:left;">
chr2:148687969-148778292
</td>
<td style="text-align:left;">
-43.9
</td>
<td style="text-align:left;">
1359-1417
</td>
<td style="text-align:left;">
5893-5951
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-2518.1
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
ENST00000409085
</td>
<td style="text-align:left;">
AAK1
</td>
<td style="text-align:left;">
chr2:69694920-69870849
</td>
<td style="text-align:left;">
-43.9
</td>
<td style="text-align:left;">
1357-1414
</td>
<td style="text-align:left;">
4140-4197
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-3785.1
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
ENST00000562343
</td>
<td style="text-align:left;">
RP11-923I11.6
</td>
<td style="text-align:left;">
chr12:52211684-52213934
</td>
<td style="text-align:left;">
-43.5
</td>
<td style="text-align:left;">
1347-1406
</td>
<td style="text-align:left;">
1901-1960
</td>
<td style="text-align:left;">
ncRNA
</td>
<td style="text-align:left;">
-493.7
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
ENST00000361183
</td>
<td style="text-align:left;">
CAPZA2
</td>
<td style="text-align:left;">
chr7:116502528-116562103
</td>
<td style="text-align:left;">
-43.3
</td>
<td style="text-align:left;">
1348-1410
</td>
<td style="text-align:left;">
4122-4184
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-1903.7
</td>
<td style="text-align:left;color: red !important;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
<td style="text-align:left;">
minEnergy
</td>
<td style="text-align:left;">
ENST00000436290
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="c.-additionally-output-always-has-a-list-result_lnc_rnatorna_list" class="section level5">
<h5><strong>C. additionally output always has a list “result_lnc_RNAtoRNA_list”</strong></h5>
<p>This list has complete results (including binding sites, location, rank) for each of the analyzed transcripts. Each transcript has assigned two sub-lists “ENSTnumber_minEnergy” and “ENSTnumber_sumEnergy” which were used for selection of the top targets. Those dataframes are a raw otput from RNARNA database.</p>
<div style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:200px; overflow-x: scroll; width:100%; ">
<table class="table table-condensed" style="font-size: 8px; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Rank
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Ensemble_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Position_hg19
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
MinEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_query
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Binding_site_target
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
location
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
SumEnergy
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_id
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
ENST00000607599
</td>
<td style="text-align:left;">
CHURC1
</td>
<td style="text-align:left;">
chr14:65381080-65402086
</td>
<td style="text-align:left;">
-60.6
</td>
<td style="text-align:left;">
775-836
</td>
<td style="text-align:left;">
1206-1267
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-1678.9
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
ENST00000373833
</td>
<td style="text-align:left;">
RCC1
</td>
<td style="text-align:left;">
chr1:28832570-28865812
</td>
<td style="text-align:left;">
-50.4
</td>
<td style="text-align:left;">
1354-1417
</td>
<td style="text-align:left;">
2332-2395
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-753.8
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
ENST00000599531
</td>
<td style="text-align:left;">
PNMAL2
</td>
<td style="text-align:left;">
chr19:46994449-46999755
</td>
<td style="text-align:left;">
-49.0
</td>
<td style="text-align:left;">
1353-1404
</td>
<td style="text-align:left;">
3518-3569
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-807.4
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
ENST00000329047
</td>
<td style="text-align:left;">
SEPT9
</td>
<td style="text-align:left;">
chr17:75315598-75496674
</td>
<td style="text-align:left;">
-46.0
</td>
<td style="text-align:left;">
1359-1408
</td>
<td style="text-align:left;">
701-750
</td>
<td style="text-align:left;">
UTR5
</td>
<td style="text-align:left;">
-762.7
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
ENST00000507938
</td>
<td style="text-align:left;">
AC090587.2
</td>
<td style="text-align:left;">
chr11:3837983-3843130
</td>
<td style="text-align:left;">
-45.3
</td>
<td style="text-align:left;">
1369-1416
</td>
<td style="text-align:left;">
846-893
</td>
<td style="text-align:left;">
ncRNA
</td>
<td style="text-align:left;">
-563.6
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
ENST00000382764
</td>
<td style="text-align:left;">
CYorf17
</td>
<td style="text-align:left;">
chrY:23544841-23548246
</td>
<td style="text-align:left;">
-44.9
</td>
<td style="text-align:left;">
1370-1419
</td>
<td style="text-align:left;">
690-739
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-240.1
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
ENST00000392857
</td>
<td style="text-align:left;">
ORC4
</td>
<td style="text-align:left;">
chr2:148687969-148778292
</td>
<td style="text-align:left;">
-43.9
</td>
<td style="text-align:left;">
1359-1417
</td>
<td style="text-align:left;">
5893-5951
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-2518.1
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
ENST00000409085
</td>
<td style="text-align:left;">
AAK1
</td>
<td style="text-align:left;">
chr2:69694920-69870849
</td>
<td style="text-align:left;">
-43.9
</td>
<td style="text-align:left;">
1357-1414
</td>
<td style="text-align:left;">
4140-4197
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-3785.1
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
ENST00000562343
</td>
<td style="text-align:left;">
RP11-923I11.6
</td>
<td style="text-align:left;">
chr12:52211684-52213934
</td>
<td style="text-align:left;">
-43.5
</td>
<td style="text-align:left;">
1347-1406
</td>
<td style="text-align:left;">
1901-1960
</td>
<td style="text-align:left;">
ncRNA
</td>
<td style="text-align:left;">
-493.7
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
ENST00000361183
</td>
<td style="text-align:left;">
CAPZA2
</td>
<td style="text-align:left;">
chr7:116502528-116562103
</td>
<td style="text-align:left;">
-43.3
</td>
<td style="text-align:left;">
1348-1410
</td>
<td style="text-align:left;">
4122-4184
</td>
<td style="text-align:left;">
UTR3
</td>
<td style="text-align:left;">
-1903.7
</td>
<td style="text-align:left;">
List of RNAs interacting with ENST00000436290(MIR4500HG)
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="copying-gene-list-to-the-clipboard-with-venny.list" class="section level3">
<h3>13. Copying gene list to the clipboard with venny.list()</h3>
<p>If you want to quickly check the overlaps betwen the gene lists you can utilize function venny.list() it will copy to clipboard the vector of characters, remove NAs and duplicates. Additionally it will print the number of collected items. This function was created to use together with the webpage <a href="http://bioinfogp.cnb.csic.es/tools/venny/" class="uri">http://bioinfogp.cnb.csic.es/tools/venny/</a> or paste results to the excel or cytoscape</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1"></a>genes&lt;-<span class="kw">c</span>(<span class="st">&#39;SGCD&#39;</span>, <span class="st">&#39;TBC1D8B&#39;</span>, <span class="st">&#39;PTGER3&#39;</span>, <span class="st">&#39;ANO5&#39;</span>, <span class="st">&#39;FGFR1&#39;</span>, <span class="st">&#39;PTGFR&#39;</span>,<span class="st">&#39;SNTB2&#39;</span>, <span class="st">&#39;USP9Y&#39;</span>, <span class="st">&#39;PTPRM&#39;</span>, <span class="st">&#39;CRIM1&#39;</span>, <span class="st">&#39;BRD1&#39;</span>, <span class="st">&#39;RAD23B&#39;</span>, <span class="st">&#39;PRKDC&#39;</span>)</span>
<span id="cb19-2"><a href="#cb19-2"></a>counts&lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">66</span>, <span class="dv">62</span>, <span class="dv">54</span>, <span class="dv">40</span>, <span class="dv">34</span>, <span class="dv">32</span>, <span class="dv">32</span>, <span class="dv">16</span>, <span class="dv">15</span>, <span class="dv">15</span>, <span class="dv">15</span>, <span class="dv">14</span>,<span class="dv">10</span>)</span>
<span id="cb19-3"><a href="#cb19-3"></a>input&lt;-<span class="st"> </span><span class="kw">data.frame</span>(genes,counts)</span>
<span id="cb19-4"><a href="#cb19-4"></a></span>
<span id="cb19-5"><a href="#cb19-5"></a>wizbionet<span class="op">::</span><span class="kw">venny.list</span>(input<span class="op">$</span>genes)</span></code></pre></div>
</div>
</div>



<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
