<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>lncRNA targets scrapper — lnc_RNARNA_scanner • wizbionet</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="lncRNA targets scrapper — lnc_RNARNA_scanner" />
<meta property="og:description" content="This function identify targets of lncRNA by automatized mining of the
http://rtools.cbrc.jp/cgi-bin/RNARNA/index.pl  database using ENST ids.
As input for target prediction, function uses ENST transcript IDs (ENST_input)
and downloads top 500 (nr_top_genes) of the results with highest i) sumenergy ii) minenergy directly into R.
After identification of the lncRNAs it will check target overlap with our DE genes.
Finally, it filters out DE genes and combined obtained results with your expression data set/interesting transcripts(ENST_targets)

" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">wizbionet</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/vignette_wizbionet.html">vignette-wizbionet</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/wizbionet/wizbionet/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>lncRNA targets scrapper</h1>
    
    <div class="hidden name"><code>lnc_RNARNA_scanner.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function identify targets of lncRNA by automatized mining of the
http://rtools.cbrc.jp/cgi-bin/RNARNA/index.pl  database using ENST ids.
As input for target prediction, function uses ENST transcript IDs (ENST_input)
and downloads top 500 (nr_top_genes) of the results with highest i) sumenergy ii) minenergy directly into R.
After identification of the lncRNAs it will check target overlap with our DE genes.
Finally, it filters out DE genes and combined obtained results with your expression data set/interesting transcripts(ENST_targets)</p>


    </div>

    <pre class="usage"><span class='fu'>lnc_RNARNA_scanner</span>(<span class='no'>ENST_input</span>, <span class='no'>analyze_onlyDEtargets</span>, <span class='no'>ENST_targets</span> ,<span class='no'>nr_top_genes</span>)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>ENST_input</th>
      <td><p>vector of characters (human ENST transcript numbers).
RNARNAdb will show information during the analysis if non coding transcript was present in the database. Careful, it sometimes also recognizes miRNAs.</p></td>
    </tr>
    <tr>
      <th>analyze_onlyDEtargets</th>
      <td><p>TRUE or FALSE (logical), it says if you want to check all possible lncRNA targets or only DE targets.</p></td>
    </tr>
    <tr>
      <th>ENST_targets</th>
      <td><p>vector of characters of human ENST numbers of your interest, usually DE mRNAs</p></td>
    </tr>
    <tr>
      <th>nr_top_genes</th>
      <td><p>numeric. Cutoff number for RNARNA database top targets output,
default set by us is 500. From this number function willextract your targets of interest</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>output is a list of data frames.</p>
<p>#Depending if analyze_onlyDEtargets= was set as TRUE
the output will be include list "lncRNA_DEtargets" which has only top DE targets of analyzed lncRNAs</p>


<p>#Depending if analyze_onlyDEtargets= was set as FALSE
the output will be include list "lncRNA_AllTargets" which has all possible top targets of analyzed lncRNAs</p>
<p>those two outputs have column ""full_id" which describes interacting RNAs with your lncRNAs
and provide a full name of the lncRNA for example: "List of RNAs interacting with ENST00000423793(RP11-432J22.2)".</p>
<p>You can use this column  for aggregating using col_agrecounter() and than clusterizer_oneR()
to obtain top lncRNAs or top targets.</p>



<p>#additionally output always has a list "result_lnc_RNAtoRNA_list"
which has complete results (including binding sites, location, rank) for each of the analyzed transcripts.
Each transcript has assigned two sub-lists "ENSTnumber_minEnergy" and "ENSTnumber_sumEnergy"
which were used for selection of the top targets</p>
<p>To access the lists use $ sign or[[]]</p>
    <h2 class="hasAnchor" id="references"><a class="anchor" href="#references"></a>References</h2>

    <p>RNARNA.db creators: Terai G, Iwakiri J, Kameda T, Hamada M, Asai K.
Comprehensive prediction of lncRNA-RNA interactions in human transcriptome. BMC Genomics. 2016;17 Suppl 1(Suppl 1):12. Published 2016 Jan 11. doi:10.1186/s12864-015-2307-5
 http://rtools.cbrc.jp/cgi-bin/RNARNA/index.pl</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>#Example</span>
<span class='co'>#requires(XML)</span>
<span class='no'>ENST_input</span><span class='kw'>&lt;-</span><span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>'ENST00000436290'</span>, <span class='st'>'ENST00000432892'</span>, <span class='st'>'ENST00000454526'</span>,
<span class='st'>'ENST00000423793'</span>, <span class='st'>'ENST00000436960'</span>,<span class='st'>'ENST00000412754'</span>
)

<span class='no'>ENST_targets</span><span class='kw'>&lt;-</span><span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>'ENST00000491143'</span>, <span class='st'>'ENST00000314191'</span>,
<span class='st'>'ENST00000309733'</span>, <span class='st'>'ENST00000377122'</span>, <span class='st'>'ENST00000338981'</span>,
<span class='st'>'ENST00000495827'</span>, <span class='st'>'ENST00000425708'</span>, <span class='st'>'ENST00000272748'</span>,
<span class='st'>'ENST00000336278'</span>, <span class='st'>'ENST00000282466'</span>, <span class='st'>'ENST00000260197'</span>,
<span class='st'>'ENST00000265354'</span>, <span class='st'>'ENST00000422989'</span>, <span class='st'>'ENST00000280527'</span>,
<span class='st'>'ENST00000342292'</span>, <span class='st'>'ENST00000426548'</span>, <span class='st'>'ENST00000340134'</span>,
<span class='st'>'ENST00000396930'</span>, <span class='st'>'ENST00000373510'</span>, <span class='st'>'ENST00000262971'</span>,
<span class='st'>'ENST00000341809'</span>, <span class='st'>'ENST00000357242'</span>, <span class='st'>'ENST00000271417'</span>,
<span class='st'>'ENST00000323816'</span>, <span class='st'>'ENST00000392806'</span>, <span class='st'>'ENST00000238682'</span>,
<span class='st'>'ENST00000398485'</span>, <span class='st'>'ENST00000422280'</span>, <span class='st'>'ENST00000376223'</span>, <span class='st'>'ENST00000237623'</span>)
<span class='no'>nr_top_genes</span><span class='kw'>=</span><span class='fl'>500</span>
<span class='no'>analyze_onlyDEtargets</span><span class='kw'>=</span><span class='fl'>FALSE</span>
<span class='no'>output</span><span class='kw'>&lt;-</span> <span class='fu'>lnc_RNARNA_scanner</span> (<span class='no'>ENST_input</span>, <span class='no'>analyze_onlyDEtargets</span>,
<span class='no'>ENST_targets</span> ,<span class='no'>nr_top_genes</span>)</div><div class='output co'>#&gt; [1] "Input explanation:"
#&gt; [1] "1. ENST_input            = ENST transcript numbers. RNARNAdb will automaticly recognize lncRNAs"
#&gt; [1] "2. analyze_onlyDEtargets = logical, do you want to check all possible targets or only DE targets?"
#&gt; [1] "3. ENST_targets = ENST numbers of targets of your interest, usually DE mRNAs"
#&gt; [1] "4. nr_top_genes          = number of top targets from RNARNA databse output"
#&gt; [1] "ENST00000436290; transcript: 1 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000432892; transcript: 2 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000454526; transcript: 3 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000423793; transcript: 4 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000436960; transcript: 5 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000412754; transcript: 6 from 6 | was found in database?: yes"
#&gt; [1] "ENST00000436290; number: 1 from 6"
#&gt; [1] "ENST00000436290_results; number: 1 from 6"
#&gt; [1] "ENST00000432892; number: 2 from 6"
#&gt; [1] "ENST00000432892_results; number: 2 from 6"
#&gt; [1] "ENST00000454526; number: 3 from 6"
#&gt; [1] "ENST00000454526_results; number: 3 from 6"
#&gt; [1] "ENST00000423793; number: 4 from 6"
#&gt; [1] "ENST00000423793_results; number: 4 from 6"
#&gt; [1] "ENST00000436960; number: 5 from 6"
#&gt; [1] "ENST00000436960_results; number: 5 from 6"
#&gt; [1] "ENST00000412754; number: 6 from 6"
#&gt; [1] "ENST00000412754_results; number: 6 from 6"
#&gt; [1] "Please cite creators of RNARNA database: doi: 10.1186/s12864-015-2307-5"
#&gt; [1] "Please cite creator of this package: Wicik Z."
#&gt; [1] "suggested next step: use function col_agrecounter"
#&gt; [1] "lncRNA_AllTargets"        "result_lnc_RNAtoRNA_list"</div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span>(<span class='no'>output</span>)</div><div class='output co'>#&gt; [1] "lncRNA_AllTargets"        "result_lnc_RNAtoRNA_list"</div><div class='input'>

</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Zofia Wicik.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


